client/packages/requisitions/src/ResponseRequisition/DetailView/IndicatorsTab.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import {
+  Box,
+  ButtonWithIcon,
+  IndicatorLineRowNode,
+  NothingHere,
+  PlusCircleIcon,
+  useTranslation,
+} from '@openmsupply-client/common';
+import { ProgramIndicatorFragment, ResponseFragment } from '../api';
+
+interface IndicatorTabProps {
+  onClick: (
+    programIndicator?: ProgramIndicatorFragment,
+    indicatorLine?: IndicatorLineRowNode,
+    response?: ResponseFragment
+  ) => void;
+  isLoading: boolean;
+  response?: ResponseFragment;
+  indicators?: ProgramIndicatorFragment[];
+}
+
+export const IndicatorsTab = ({
+  onClick,
+  isLoading,
+  response,
+  indicators,
+}: IndicatorTabProps) => {
+  const t = useTranslation();
+  if (isLoading) {
+    return <NothingHere body={t('error.no-indicators')} />;
```

Comment:
@roxy-dao shouldn't this be a loading state page rather than a no indicators page?

Reply:
Yeah XD